src/System.Data.OleDb/ref/System.Data.OleDb.cs

```diff
@@ -244,7 +239,7 @@ public sealed partial class OleDbErrorCollection : System.Collections.ICollectio
     public sealed partial class OleDbException : System.Data.Common.DbException
     {
         internal OleDbException() { }
-        public override int ErrorCode { get { throw null; } }
```

Comment:
Why was this removed?  Because OleDbException is sealed?

Reply:
It's moved to the manual file.